### DISCUSSION TITLE
Differential gene expression between 10X visium spatial transcriptomics samples

### DISCUSSION THREAD

**Discussion by rebecca-ong** [satijalab/seurat#6008] [Q&A]
Hi everyone,

I currently have some 10X visium spatial transcriptomics data that I am looking to perform differential gene expression analysis on in different regions of the tissue (mouse brain) between each of my samples (i.e., condition vs. control). I wanted to ask whether anyone has done something similar in terms of performing differential expression analysis across visium samples for specific regions in the tissue using only Seurat or if it requires integration of different analysis tools (i.e., DESeq2) in order to do that. I've seen that sctransform can be used for single cell data but I was curious how people have used it for spatial transcriptomics data.

I would love to hear any thoughts or if you would be able to guide me to any resources!

Cheers,
Rebecca

**Comment by vucinick** [comment]
Hi Rebecca,

I'm fairly new in this type of analysis and I have been also trying to find how to do DGE on spatial transcriptomics data between samples (condition vs. control). 
I was wondering how did you end up doing it? 
I am planning to create a pseudo-bulk samples as mention in [OSCA multi-sample - chapter 4.3](http://bioconductor.org/books/3.15/OSCA.multisample/multi-sample-comparisons.html#creating-pseudo-bulk-samples) and run the DEA, but I am not sure if it's a proper way.

Best,
Kim

**Reply by rebecca-ong** [reply]
Hi Kim,

Thank you for your reply and I ended up using the functions that @carolgirardi mentioned, described in this vignette [Differential expression testing](https://satijalab.org/seurat/archive/v3.0/de_vignette.html). I also asked around for advice from other bioinformaticians at my institute and this vignette: [Integrating stimulating vs. control PBMC datasets to learn cell-type specific responses](https://satijalab.org/seurat/archive/v3.0/immune_alignment.html) , was also mentioned to me and one that I found rather helpful for my analysis. I hope that helps in some way!

I'm also new to all these types of analyses and because ST is also so new, it seems to be a general consensus that there hasn't really been a set pipeline established to perform differential expression analysis between spatial samples. My analysis is still ongoing and I'm currently building my sample sizes so I have progressed much in terms of playing around with my analysis as I'm waiting for the rest of my samples to be done.

Cheers,
Rebecca

**Reply by vucinick** [reply]
Hi Rebecca,

This is very helpful, thank you! I'm glad you are nicely progressing :) 
After reading and going through multiple discussion, I've decided to go with FindMarkers() function but I am still trying to figure out which test would be the best for my data. I was just wondering why did @carolgirardi (and you - if you used it) decide to use MAST, and not DESeq2 or some other test?

If someone else comes across this post, here are my two cents after reading multiple threads. The assay that you use is important. If your samples have comparable sequencing depths, then you can use SCT assay for DE between samples/conditions/etc, otherwise you should use RNA assay. Reference: #5047 and #5009.

Cheers,
Kim

**Comment by carolgirardi** [comment]
Hi Rebecca and Kim
I used the same pipeline commonly used for sc data: SCT normalization and FindMarkers() with MAST as argument.
I am also new to visium analysis, so any comments are welcome :)
Best,
Carol
